### PR TITLE
compute the preflight_operators_to_certify var

### PR DESCRIPTION
**Pull request by fredericlepied**
Depends-On: https://github.com/dci-labs/dallas-pipelines/pull/779
Depends-On: https://github.com/openshift-kni/example-cnf/pull/7

### CODE REVIEW

File: testpmd/hooks/pre-run.yml [hunk 1]
```diff
@@ -73,6 +74,41 @@
   when:
     - dci_change_dir is defined
 
+- name: Create the preflight_operators_to_certify variable
+  when:
+    - certify_operators is defined
+    - certify_operators | bool
```
Comment (tkrishtop) [inline testpmd/hooks/pre-run.yml:80]: Hi Fred, you could write it shorter
```
when:
  - certify_operators | default(false) | bool
  - example_cnf_index_image is defined 
```

**Review by tkrishtop — APPROVED** [review]

**Comment by fredericlepied** [comment]
check workload example-cnf example-cnf:ansible_tags=kubeconfig,pre-run,redhat-testing,post-run,success example-cnf:ansible_skip_tags=sriov example-cnf:ansible_extra_vars=do_cnf_cert:false

**Comment by fredericlepied** [comment]
check workload example-cnf example-cnf:ansible_tags=kubeconfig,job,pre-run,redhat-testing,post-run,success example-cnf:ansible_skip_tags=sriov example-cnf:ansible_extra_vars=do_cnf_cert:false

File: testpmd/hooks/pre-run.yml [hunk 2]
```diff
@@ -73,6 +74,43 @@
...
+    - certify_operators | default(false) | bool
+    - example_cnf_index_image is defined
+  block:
+    - name: Get Catalog Data
+      ansible.builtin.shell:
+        cmd: >
+          set -e -o pipefail;
+          ~/clusterconfigs-{{ cluster_name }}/opm render
+          {% if mc_pullsecret is defined %}
+          --authfile {{ mc_pullsecret }}
```
Comment (capolrik) [inline testpmd/hooks/pre-run.yml:88]: I got an error on this cmd, --authfile is not a valid option

**Comment by fredericlepied** [comment]
check workload example-cnf example-cnf:ansible_tags=kubeconfig,job,pre-run,redhat-testing,post-run,success example-cnf:ansible_skip_tags=sriov example-cnf:ansible_extra_vars=do_cnf_cert:false

**Comment by fredericlepied** [comment]
check workload example-cnf example-cnf:ansible_tags=kubeconfig,job,pre-run,redhat-testing,post-run,success example-cnf:ansible_skip_tags=sriov example-cnf:ansible_extra_vars=do_cnf_cert:false

**Pull request merged**